arcgis-ios-sdk-samples/Cloud and portal/Search for webmap by keyword/SearchForWebmapByKeywordViewController.swift

```diff
@@ -38,16 +38,6 @@ class SearchForWebmapByKeywordViewController: UICollectionViewController {
         
         (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["SearchForWebmapByKeywordViewController", "WebMapCell", "WebMapViewController"]
         
-        if #available(iOS 13, *) {
-            // for iOS 13 this must be added in viewDidLoad
-            // for iOS 12, it gets added in viewDidAppear, because for iOS 12 it doesn't
-            // show up if added here.
-            addSearchController()
-        }
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
         if navigationItem.searchController == nil {
```

Comment:
This check is no longer needed at `navigationItem.searchController` will always be `nil` when this method (`viewDidLoad()`) is called.